### PR TITLE
Payload encrypt decrypt

### PR DESCRIPTION
**Pull request by bgfoshay**
## What’s changing?

Update APIM to encrypt payloads and Bulk Upload function to decrypt.

## Why?

NAC-605 - Bulk Upload Payload Encryption/Decryption

## This PR has:

- [x] **Commented source code** (_required on public classes and methods, and elsewhere as appropriate_)

- [ ] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

- [x] **Automated unit tests** (_to maintain or increase level of code coverage_)

- [x] **Changes to IAC** (_add any deployment steps that require manual intervention to the draft release notes_)

### CODE REVIEW

File: etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs [hunk 1]
```diff
@@ -109,7 +113,7 @@ public void GetBlob_TestBlob()
 
             // Arrange
             var blobClientStream = new BlobClientStream();
-            Environment.SetEnvironmentVariable("BlobStorageConnectionString", "UseDevelopmentStorage=true");
+            Environment.SetEnvironmentVariable("BlobStorageConnectionString", "DefaultEndpointsProtocol=https;EndpointSuffix=core.windows.net;AccountName=test;AccountKey=dr60GUR5mGtodn3KTot4sLIYzB8qKbs9z5Ync5lVy0hOtnr4UFf/mjGFTHhT+Ef0hfGr5kmqjHCq+AStap+0ZA==");
```
Comment (dborgen2) [inline etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs:116]: Question: Just double checking this is supposed to be here. It looks like a Key?

File: etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 using Azure.Messaging.EventGrid.SystemEvents;
-using Azure.Storage;
-using Azure.Storage.Blobs;
 using Azure.Storage.Blobs.Models;
 using Azure.Storage.Blobs.Specialized;
 using Microsoft.Extensions.Logging;
 using Moq;
 using Xunit;
 using System.Text.Json;
+using System.Text;
```
Comment (dborgen2) [inline etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs:13]: Nitpick: Looks like your system might not have the right settings for ordering these usings either. Let me know if you want me to tell you what I had to change

Comment (bgfoshay) [reply etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs:13]: Dang, you're right. Good catch.

File: etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs (hunk 1 above)
Comment (bgfoshay) [reply etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs:116]: It's a fake key. Thanks for checking though.

File: etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs (hunk 2 above)
Comment (bgfoshay) [reply etl/tests/Piipan.Etl.Func.BulkUpload.Tests/Parsers/BlobClientStreamTests.cs]: Actually, turns out I had the setting, but never applied the formatting. Should be fixed now.

**Review by dborgen2 — APPROVED** [review]
👍

**Pull request merged**